packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -742,6 +770,12 @@ describe('AddresslistVoting', function () {
         expect((await voting.getProposal(id)).tally.yes).to.equal(0);
         expect((await voting.getProposal(id)).tally.no).to.equal(0);
         expect((await voting.getProposal(id)).tally.abstain).to.equal(1);
+
+        await voting.vote(id, VoteOption.None, false);
+        await voting.vote(id, VoteOption.None, false);
+        expect((await voting.getProposal(id)).tally.yes).to.equal(0);
```

Comment:
Don't we need to await the tx being fully mined with HardHat? (speaking from experience with ethers.js)

Reply:
https://hardhat.org/hardhat-network/docs/explanation/mining-modes
> Hardhat Network can be configured to automine blocks, immediately upon receiving each transaction,
> [...]
> By default, only the automine mode is enabled.